docs/releases/5.2.txt

```diff
@@ -269,7 +268,8 @@ Models
   specified order of the referenced expressions. Previously the order was based
   of a set of counterintuitive rules which made query combination through
   methods such as
-  :meth:`QuerySet.union()<django.db.models.query.QuerySet.union>` unpredictable.
+  :meth:`QuerySet.union()<django.db.models.query.QuerySet.union>`
+  unpredictable.
```

Comment:
I'd change this to
```
  methods such as :meth:`.QuerySet.union` unpredictable.
```

Reply:
Thanks, good advice, applying and merging! :rocket: